src/styles/index.scss

```diff
@@ -1,3 +1,143 @@
+:root {
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
```

Comment:
This is valid, but why not using scss variables here?

In addition I would suggest you cleaner names:
main-accent -> main-accent-color, 
secondary (font-size? color? padding?) -> secondary-color
blue-accent (what if designers will change it to orange?) -> moyo-blue (ok, as it refers to brand book), secondary-accent-color, link-acent-color